OpenRA.Mods.Common/EditorBrushes/EditorActorBrush.cs

```diff
@@ -50,10 +51,15 @@ public EditorActorBrush(EditorViewportControllerWidget editorWidget, ActorInfo a
 			if (buildingInfo != null)
 				centerOffset = buildingInfo.CenterOffset(world);
 
+			// Enforce Neutral or Creeps owner if the actor has RequiresSpecificOwner
+			var specificOwnerInfo = actor.TraitInfoOrDefault<RequiresSpecificOwnerInfo>();
+			if (specificOwnerInfo != null && !specificOwnerInfo.ValidOwnerTypes.Contains(ownerName))
+				ownerName = specificOwnerInfo.ValidOwnerTypes.First();
```

Comment:
I wonder if it is worth to log this (in e.g. `debug.log`). Or even throw an exception instead of silently failing.

Reply:
This is silently correcting the error. An exception could mean loosing all mapping progress, which would disturb things way too much.